imap/httpd.h

```diff
@@ -185,7 +183,6 @@ struct auth_scheme_t {
 /* Auth scheme identifiers */
 enum {
     AUTH_BASIC        = (1<<0),
-    AUTH_DIGEST       = (1<<1),
     AUTH_SPNEGO       = (1<<2),
```

Comment:
Can you update this PR to either add a comment that (1<<1) was deprecated, or adjust the values for AUTH_SPNEGO, etc?  Having values of the enum jump from (1<<0) to (1<<2) with now explanation would look weird.